studio/queues/rabbit_queue.py

```diff
@@ -68,11 +70,13 @@ def __init__(self, queue, route, amqp_url='',
 
         self._queue = queue
         self._queue_deleted = True
+        self._connection_failed = False
+        self._connection_failure_reason = None
 
```

Comment:
Status flag indicating whether "connect" operation on RMQ have succeeded or not.
 _connection_failure_reason will contain a string returned by lower-level exception.